pkg/version/version.go

```diff
@@ -45,10 +45,6 @@ func GetOpenShiftVersion() string {
 	return openshiftVersion
 }
 
-func GetOpenShiftSemverVersion() (semver.Version, error) {
-	return semver.Make(strings.TrimPrefix(GetOpenShiftVersion(), VersionPrefix))
-}
-
```

Comment:
+1, In fact I once missed it. I think it was once used for the auto update which we for now disabled.